x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/new_terms/README.md

```diff
@@ -27,4 +27,3 @@ The new terms rule type reuses the singleSearchAfter function which implements t
 ## Limitations and future enhancements
 
 - Value list exceptions are not supported at the moment. Commit ead04ce removes an experimental method I tried for evaluating value list exceptions.
-- Runtime field supports only 100 emitted values. So for large arrays or combination of values greater than 100, results may not be exhaustive. This applies only to new terms with multiple fields
```

Comment:
This limitation could still be hit, right? It's just less likely now

Reply:
Yes, correct. But we also have limitations of 100 alerts. So essentially, it won't make any difference if we hit it here or during alerts generations. 
But probably worth to leave it anyway. It's an internal doc, so might be useful in future